lib/rails_admin/config/actions/base.rb

```diff
@@ -78,6 +78,11 @@ class Base
           true
         end
 
+        # Target window [_self, _blank]
+        register_instance_option :target_window do
+          :_self
```

Comment:
Could we default to no attribute instead? Maybe that would be handled by `nil`? In the normal cause I wouldn't expect `target="_self"`, just nothing.

Reply:
Removed the default value, however, I'm not sure what's the test that can cover this change;
Any idea ?

Reply:
>  Any idea ?

Would id be possible to write a test that renders HTML with this option? This HTML either contains the attribute `target="_blank"` or it doesn't.